src/gc/gceventstatus.h

```diff
@@ -29,7 +29,7 @@
 #include "gc.h"
 
 // Uncomment this define to print out event state changes to standard error.
-// #define TRACE_GC_EVENT_STATE 1
+#define TRACE_GC_EVENT_STATE 1
```

Comment:
Should this line be commented out?

Reply:
you're right, thanks - missed this from debugging earlier.